src/main/java/RaiseTechTask/TaskStudentsManagement_final/version/entity/Courses.java

```diff
@@ -0,0 +1,15 @@
+package RaiseTechTask.TaskStudentsManagement_final.version.entity;
+
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+public class Courses {
```

Comment:
コース単体を表すので単数形がbetter 👍🏽 
```suggestion
public class Course {
```

Reply:
Studentsも同様。
ただしテーブル名は複数形のままでよいです！

Reply:
命名に関してはもう少し慎重に考えるように意識していこうと思います。
なんとなくや細かいところまで考えてつけられていませんでした。